src/views/DatacenterBrowser/Progress.js

```diff
@@ -1,9 +1,15 @@
 import m from "mithril";
+import moment from "moment";
+
 
 const deviceToProgress = device => {
+    console.log(device);
```

Comment:
Might want to lose the log :)

Reply:
I got so excited by it working I missed a few things.

Reply:
fixed in 478b105